pages/api/journal/get.ts

```diff
@@ -0,0 +1,18 @@
+import { NextApiResponse, NextApiRequest } from "next";
+import {getJournalEntriesByReviewStatus} from "server/actions/Contentful";
+//This API route takes in a query from the url. By default, the route will return all entries that have not been reviewed. To get reviewed entries, use the following route: /api/journal/get?reviewed=true
+export default async function handler(req: NextApiRequest, res: NextApiResponse){
+
+    if(req.method == "GET"){
+        let reviewed = false;
+        if(req.query.reviewed){
+           reviewed = (req.query.reviewed === "true") || (req.query.reviewed === "false");
+        }
```

Comment:
This bit is a little confusing. Don't we want to call getJournalEntriesByReviewStatus() with req.query.reviewed?

Reply:
req.query.reviewed is a string, and the data type within Contentful is a boolean. So in order to retrieve posts by review status, req.query.reviewed has to be converted into a boolean.  I just realized that I messed up the assignment for reviewed, so I'll fix that really quick.